website/content/api-docs/health.mdx

```diff
@@ -259,6 +259,8 @@ The table below shows this endpoint's support for
 - `filter` `(string: "")` - Specifies the expression used to filter the
   queries results prior to returning the data.
 
+- `peer` `(string: "")` - Specifies the peer from which the service was imported.
```

Comment:
is this accurate?

```suggestion
- `peer` `(string: "")` - Specifies the imported service's peer.
```

Reply:
I wanted to discuss this actually; a service is either "imported" or local; the current wording suggests the service is definitively imported but I'm not sure how to convey that in a more conditional way.

Maybe appending "Applies only to imported services"?